app/mesh/1.0.x/installation/docker.md

```diff
@@ -26,15 +26,15 @@ You have a license for {{site.mesh_product_name}}.
 {{site.mesh_product_name}} provides the following Docker images for all of its
 executables:
 
-* **kuma-cp**: at `kong-docker-kong-mesh-docker.bintray.io/kuma-cp:{{page.kong_versions[0].version}}`
-* **kuma-dp**: at `kong-docker-kong-mesh-docker.bintray.io/kuma-dp:{{page.kong_versions[0].version}}`
-* **kumactl**: at `kong-docker-kong-mesh-docker.bintray.io/kumactl:{{page.kong_versions[0].version}}`
-* **kuma-prometheus-sd**: at `kong-docker-kong-mesh-docker.bintray.io/kuma-prometheus-sd:{{page.kong_versions[0].version}}`
+* **kuma-cp**: at `docker.io/kong/kuma-cp:{{page.kong_versions[0].version}}`
+* **kuma-dp**: at `docker.io/kong/kuma-dp:{{page.kong_versions[0].version}}`
+* **kumactl**: at `docker.io/kong/kumactl:{{page.kong_versions[0].version}}`
+* **kuma-prometheus-sd**: at `docker.io/kong/kuma-prometheus-sd:{{page.kong_versions[0].version}}`
```

Comment:
Since these images are now public, I recommend turning these into links to Docker Hub instead of just code snippets. Eg:

```suggestion
* **kuma-cp**: at [`docker.io/kong/kuma-cp:{{page.kong_versions[0].version}}`](https://hub.docker.com/r/kong/kuma-cp)
* **kuma-dp**: at [`docker.io/kong/kuma-dp:{{page.kong_versions[0].version}}`](https://hub.docker.com/r/kong/kuma-dp)
* **kumactl**: at [`docker.io/kong/kumactl:{{page.kong_versions[0].version}}`](https://hub.docker.com/r/kong/kumactl)
* **kuma-prometheus-sd**: at [`docker.io/kong/kuma-prometheus-sd:{{page.kong_versions[0].version}}`](https://hub.docker.com/r/kong/kuma-prometheus-sd)
```

You can also link the image names instead of the full path, up to you.